### ISSUE TITLE
AttributeError: module 'nasdaq_stock' has no attribute 'stock'

### ISSUE THREAD

**Issue by zurgeg** [georgejs/nasdaq_stock#1]
I was using this module in Google Colab (ipython) when this happened:
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-4-15e276ba5899> in <module>()
      1 import nasdaq_stock
      2 ticker = 'aapl'
----> 3 stock_data = nasdaq_stock.stock(ticker)

AttributeError: module 'nasdaq_stock' has no attribute 'stock'
```
Anyway to fix this??
Thanks in advance

**Comment by georgejs** [comment]
Hi Zurgeg,
It's been a while since I contributed to this project. There could have been some changes from the source.

I will spend sometime to try an correct this error.

**Comment by zurgeg** [comment]
Hi, I fix this error in PR #2. Be sure to create a new version after
pulling these changes in!

**Comment by zurgeg** [comment]
since you merged this PR i am going to close this issue!
